apps/client/src/locales/messages.nl.xlf

```diff
@@ -7762,7 +7762,7 @@
       </trans-unit>
       <trans-unit id="routes.resources.personalFinanceTools" datatype="html">
         <source>personal-finance-tools</source>
-        <target state="new">personal-finance-tools</target>
+        <target state="translated">hulpmiddelen-voor-persoonlijke-financiën</target>
```

Comment:
Could you change to `hulpmiddelen-voor-persoonlijke-financien` (without special character)?